webpack/assets/javascripts/foreman_tools.test.js

```diff
@@ -57,3 +57,51 @@ describe('deprecate', () => {
     expect(console.warn).toHaveBeenCalledWith('DEPRECATION WARNING: you are using deprecated oldtest, it will be removed in Foreman 1.42. Use tfm.tools.newtest instead.');
   });
 });
+
+/* eslint-disable max-statements */
+describe('initTypeAheadSelect', () => {
```

Comment:
@tbrisker : I believe, it's quite hard to actually test that select2 generates a select field. Imho this would actually be testing that select2 works. The test I wrote just checks, that the correct data is passed to select2 and that our callbacks work correctly.